src/renderer/components/outputs/OutputContainer.tsx

```diff
@@ -1,20 +1,23 @@
-import { Box, Center, HStack, chakra, useColorModeValue } from '@chakra-ui/react';
+import { Box, Center, HStack, Text, chakra, useColorModeValue } from '@chakra-ui/react';
 import React, { memo, useMemo } from 'react';
 import { Connection, Handle, Position, useReactFlow } from 'react-flow-renderer';
-import { useContext } from 'use-context-selector';
+import { useContext, useContextSelector } from 'use-context-selector';
 import { OutputId } from '../../../common/common-types';
 import { Type } from '../../../common/types/types';
 import { parseSourceHandle } from '../../../common/util';
 import { GlobalVolatileContext } from '../../contexts/GlobalNodeState';
 import { SettingsContext } from '../../contexts/SettingsContext';
 import { getTypeAccentColors } from '../../helpers/getTypeAccentColors';
 import { noContextMenu } from '../../hooks/useContextMenu';
+import { TypeTag } from '../TypeTag';
 
 interface OutputContainerProps {
     hasHandle: boolean;
     outputId: OutputId;
     id: string;
     definitionType: Type;
+    label: string;
+    generic: boolean;
```

Comment:
What does "generic" even mean here?

Reply:
It's basically whether or not to put the label/type props next to the handle vs above the input

Reply:
Huh, the input container does the same thing. Strange, but alright.